CHANGELOG.rst

```diff
@@ -1,3 +1,44 @@
+v1.1.2 (2023-05-25)
+-------------------
+
+New Features
+^^^^^^^^^^^^
+
+- Phase Dispersion Minimization as a method to search for periodic signals
+  in data is now implemented in the ``istingray.pulse`` submodule. To use it,
```

Comment:
Very small typo?

Reply:
Might be mine from the town crier log, coming to think of it? Oops.

Reply:
No worries, I'll fix it